lib/slack/bot.ex

```diff
@@ -38,7 +38,7 @@ defmodule Slack.Bot do
 
         url = String.to_char_list(state.rtm.url)
         {:ok, pid} = options.client.start_link(url, __MODULE__, state, [keepalive: options.keepalive])
-
+        Process.link(pid)
```

Comment:
`start_link` of `:websocket_client` should have already done this in the line above. Are you trying to solve for a different underlying client?